recipes-core/customizations-example/files/terminal_resize.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Copyright (c) Siemens AG, 2022
+#
+# Authors:
+#  Chao Zeng <chao.zeng@siemens.com>
+#
+# SPDX-License-Identifier: MIT
+
+terminal_resize() {
+    local IFS='[;' escape screen_size cols rows
+    # detect the size of the screen
+    echo -ne '\e7\e[r\e[999;999H\e[6n\e8'
+    read -t 5 -sd R escape screen_size || {
+        echo unsupported terminal emulator. >&2
+        return 1
+    }
+
+    cols="${screen_size##*;}" rows="${screen_size%%;*}"
+    if [[ "${cols}" -gt 0 && "${rows}" -gt 0 ]];then
+        stty cols "${cols}" rows "${rows}"
+    else
+        echo unsupported terminal emulator. >&2
+        return 1
+    fi
+}
+
+case $(/usr/bin/tty) in
+        /dev/ttyS3)
+        terminal_resize
+        ;;
+esac
+
```

Comment:
Extra line at EOF.

But this looks much more compact. Let's play with it and see if we may actually install it into the default profile later.

Reply:
ok, removed the extra line.